src/ceph-disk

```diff
@@ -1468,12 +1468,32 @@ def prepare_dev(
         except subprocess.CalledProcessError as e:
             raise Error(e)
 
+def check_journal_reqs(args):
+    _, allows_journal = command([
+        'ceph-osd', '--check-allows-journal',
+        '-i', '0',
```

Comment:
In theory a user could have a config setting that only applied to a 
specific id, but in practice that's not helpful since the id hasn't been 
assigned at this stage.  0 (or any other number) is fine.